packages/dialtone-vue3/components/popover/popover.vue

```diff
@@ -953,6 +968,48 @@ export default {
         onShow: this.onShow,
       });
     },
+
+    //  ============================================================================
+    //  $ HOVERCARD
+    //  ----------------------------------------------------------------------------
+
+    setInTimer () {
+      this.inTimer = setTimeout(() => {
+        this.isOpen = true;
+      }, TOOLTIP_DELAY_MS);
+    },
+
+    setOutTimer () {
+      this.outTimer = setTimeout(() => {
+        this.isOpen = false;
+      }, TOOLTIP_DELAY_MS);
+    },
+
+    onEnterAnchor (e) {
+      if (!this.hovercard) return;
+      clearTimeout(this.outTimer);
+      this.setInTimer(e);
+    },
+
+    onLeaveAnchor (e) {
+      if (!this.hovercard) return;
+      clearTimeout(this.inTimer);
+      this.setOutTimer(e);
+    },
+
+    onEnterContent () {
+      if (!this.hovercard) return;
+      clearTimeout(this.outTimer);
+    },
+
+    onLeaveContent (e) {
+      if (!this.hovercard) return;
+      this.setOutTimer(e);
+    },
+
+    //  ============================================================================
+    //  $ HOVERCARD
+    //  ----------------------------------------------------------------------------
```

Comment:
extra?

Reply:
Since this block is exclusively for hovercard behaviour I decided to use that comment section.
If this component was created in vue3 syntax, I would like to move it out to a composable like emoji picker modules.

I would like to keep that comment.

Reply:
k, wasn't sure if it was intended, sounds good.